internal/v1/server.go

```diff
@@ -135,6 +142,58 @@ func Attach(conf *ServerConfig) error {
 	})
 
 	h.server.echo.GET("/metrics", echo.WrapHandler(promhttp.Handler()))
+
+	var nonFoundRoutes []string
+
+	// as we need to implement some path registrations manually, let's check if all are there
+	fixSpecPaths := regexp.MustCompile("{(.+?)}")
+	for _, server := range spec.Servers {
```

Comment:
this part looks quite wonky, but it just mean I do not understand it by quick skim... will try to undersnad next week

Reply:
I'll wrap into a function for readability. Just maps OpenAPI syntax to internal echo-server syntax to compare them

Reply:
...not sure if this should go into a test though